ludwig/backend/ray.py

```diff
@@ -141,35 +144,109 @@ def shutdown(self):
         self.executor.shutdown()
 
 
+@ray.remote
+class MetricCollector(object):
+    def __init__(self):
+        self.metrics = []
+
+    def add_metrics(self, metrics):
+        self.metrics.append(metrics)
+
+    def collect(self):
+        return sum_dicts(
```

Comment:
This still does not convince me 100%, as most of the metrics summed / averaged across partitions do no make much sense. maybe we can deactivate evaluation also in the ray case, think it through and then reactivate it.

Reply:
I agree, though this implies it is also currently broken in the normal Horovod case as well:

https://github.com/ludwig-ai/ludwig/blob/51e357dca8b01cb3b9e687b13e1d270329c5bddf/ludwig/models/predictor.py#L289

I believe once we adopt the Metrics package for PyTorch that natively supports distributed aggregation this problem will be addressed. Until then, I agree that disabling makes sense.

Reply:
I think it depends on what we are aggregating, it it's accuracies for instance, this wouldn't be correct, it it's aggregation by summing the counts of correct predictions and all predictions and then later diving correct prediction by all predictions, this would be correct. But I believe we are doing hte formaer.